.travis.yml

```diff
@@ -4,7 +4,8 @@ python: 3.5
 sudo: false
 
 install:
-  - python3 setup.py install
+  - travis_retry pip install -e .[test]
```

Comment:
I saw it in another project's .travis.yml (which seemed like it knew what it was doing), and it seemed like a zero-cost line of defense against e.g. transient network failures and other things that shouldn't cause the build to fail. Any reason not to?